src/EditorFeatures/Core.Wpf/InlineParameterNameHints/InlineParameterNameHintsTag.cs

```diff
@@ -113,11 +113,13 @@ public async Task<IReadOnlyCollection<object>> CreateDescriptionAsync(Cancellati
                         }
                     }
                 }
+
+                var uiCollection = Implementation.IntelliSense.Helpers.BuildInteractiveTextElements(textContentBuilder.ToImmutableArray<TaggedText>(),
```

Comment:
This method requires a non-null `document` parameter, so I just moved it into the existing null check.